### DISCUSSION TITLE
Different plot size between local env and docker

### DISCUSSION THREAD

**Discussion by renato145** [quarto-dev/quarto-cli#3482] [Q&A] [answered]
Hi,

I have a docker image that runs `quarto render --execute` every couple of hours to update data and figures (I'm using python jupyter notebooks).
My problem is that while in my local environment `quarto render` produces images that fit the resulting web page, the docker image produce bigger images and the web page shows them with scrolling.
Is there any parameter I'm missing to have similar results on the docker image?

Cheers

**Comment by mcanouil** [comment]
Do you observe the same result using only Jupyter Notebook?
You might have set some settings for Jupyter in your local environment that you did not set in your Docker container.

To note, a small reproducible example would help (even yourself) figuring this out.

**Comment by renato145** [comment, marked as answer]
Got the solution, it was the default parameters for matplotlib.
For anyone having a similar problem check the differences of the `mpl.rcParams` object between your local environment and docker (`mpl` comes from `import matplotlib as mpl`).
In my case I had 3 differences so I fixed it with:
```python
diffs = {
    'figure.dpi': 72.0,
    'figure.figsize': [6.0, 4.0],
    'figure.subplot.bottom': 0.125
}
for k,v in diffs.items(): mpl.rcParams[k] = v
```
